public/styles/HomePage.scss

```diff
@@ -20,13 +20,28 @@
 
     .main {
       #primary {
+        float: left;
+        padding: 0 0 0 2em;
+        text-align: center;
+        width: 78%;
+
+        .loading-icon {
+          color: $color-accent;
+          position: relative;
+          top: 2em;
+        }
 
         .home-articles-cont {
           margin: 0 0 1em;
           overflow: visible;
 
           .card {
             height: 22em;
+            margin: 0 1% 1% 0;
+            position: relative;
+            text-align: left;
+            top: 0;
+            transition: all 0.5s;
```

Comment:
The properties from line 40 to 44 except text-align have been set in index.sccs  lines 70 - 73. So you can remove these.

Reply:
done